internal/executor/vaultunboxer/vaultunboxer.go

```diff
@@ -54,7 +56,8 @@ func NewFromEnvironment(ctx context.Context, env *environment.Environment) (*Vau
 	if jwtToken, ok := env.Lookup("CIRRUS_OIDC_TOKEN"); ok {
 		auth := &JWTAuth{
 			Token: jwtToken,
-			Role:  env.Get(EnvCirrusVaultRole),
+			Role:  env.ExpandText(env.Get(EnvCirrusVaultRole)),
```

Comment:
The `env.ExpandText()` seems to be redundant here, because the whole `env` is already expanded a couple lines before the Vault-logic kicks in:

https://github.com/cirruslabs/cirrus-ci-agent/blob/893e751a8b1f454e6abbaf7104c8fb54f66ac187/internal/executor/executor.go#L130